AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main_Optimized.py

```diff
@@ -53,6 +53,27 @@ def wrap_run(cls, instance, request, workspace, editor, editor_test_results, lau
         for f in original_file_list:
             fm._restore_file(f, file_list[f])
 
+@pytest.mark.xfail(reason="Optimized tests are experimental, we will enable xfail and monitor them temporarily.")
+@pytest.mark.SUITE_main
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestAutomationWithPrefabSystemEnabled(EditorTestSuite):
+
+    global_extra_cmdline_args = ['-BatchMode', '-autotest_mode',
+                                 'extra_cmdline_args=["--regset=/Amazon/Preferences/EnablePrefabSystem=true"]']
+
+    @staticmethod
+    def get_number_parallel_editors():
+        return 16
+
+    class C4982801_PhysXColliderShape_CanBeSelected(EditorSharedTest):
+        from .tests.collider import Collider_BoxShapeEditting as test_module
+
+    class C4982800_PhysXColliderShape_CanBeSelected(EditorSharedTest):
+        from .tests.collider import Collider_SphereShapeEditting as test_module
+
+    class C4982802_PhysXColliderShape_CanBeSelected(EditorSharedTest):
+        from .tests.collider import Collider_CapsuleShapeEditting as test_module
```

Comment:
nit: Editting -> Editing?
(Never mind If it's a pre-existing class that would take considerable effort to rename)

Reply:
Fixed.